aries-site/src/pages/components/togglegroup.mdx

```diff
@@ -0,0 +1,33 @@
+## Use Cases
+
+**Toggling Views** - Typically used in a data collection toolbar to allow the user to swtich between different data presentations such as map, list, chart and datatable.
+
+**Rich Text Editor** - Allows the user to select one or more options of related items like bold, underline and italic.
+
+
+## Anatomy
+
+| Label | Region           | Purpose           |                  Required                   | Notes             |
+| ----- | ---------------- | ----------------- | :-----------------------------------------: | ----------------- |
+| **1** | **Container** | Organizes children, how they appear and spaced. | Required |        --         |
+| **2** | **Label** | Describes what the button will do when selected. | Optional |        If no label is shown, icon is required.        |
+| **3** | **Icon** | Used to visually represent text label with or without label.  | Optional |       a11yTitle required. If no icon is shown, label is required      |
+
+
+## Types
+
+**Single select** - Allows a user to toggle on or off one option from a set of alike buttons.
+
+**Multi-select** - Allows a user to toggle on or off multiple options within a group of alike buttons at the same time.
+
+
+## Accessibility
+
+In icon-only cases, ensure each icon has a clear a11yTitle.
+
+| Key | Interaction           | 
+| ----- | ---------------- |
+| **Tab** | Focuses on first toggle button. |
```

Comment:
```suggestion
| **Tab** | If nothing is toggled on, focuses on the first button in the group. Otherwise, focuses on the first button that is toggled on. |
```